packages/eyes/src/index.ts

```diff
@@ -1,16 +1,39 @@
 /* tslint:disable:no-console */
 
+require('dotenv').config();
+const {Eyes} = require('eyes.images');
+
 import path from 'path';
 import puppeteer from 'puppeteer';
-import {WebpackConfigurator, serve, IServer, waitForPageError, logConsoleMessages} from '@ui-autotools/utils';
+import {WebpackConfigurator, serve, IServer, waitForPageError, logConsoleMessages, consoleLog} from '@ui-autotools/utils';
+import chalk from 'chalk';
+import uuid from 'uuid';
 
 const packagePath = path.resolve(__dirname, '..');
 const projectPath = process.cwd();
 const webpackConfigPath = path.join(projectPath, 'meta.webpack.config.js');
 
-function getWebpackConfig() {
+const projectName = require(path.join(projectPath, 'package.json')).name;
```

Comment:
Let's also validate this in case someone accidentally runs the script in a wrong folder.